common/config.py

```diff
@@ -46,9 +46,18 @@
 LARAVEL_HOST = getenv("LARAVEL_HOST")
 LARAVEL_ENDPOINT = "api/link-submission"
 
+TWITTER_LARAVEL_API_KEY = getenv("TWITTER_LARAVEL_API_KEY")
+
 DATE_TIME_FORMAT = "%Y-%m-%d %H:%M:%S"
 
-LAST_RUN_FILENAME = "job_last_run.json"
+TWITTER_LAST_RUN_FILENAME = "twitter_job_last_run.json"
+TWITTER_LAST_RUN_BUCKET = ""
+DEFAULT_TWITTER_LAST_RUN_ID = 1270867714505158656
```

Comment:
I don't think this is being imported anywhere

Reply:
true, more for future reference if you need for local testing